assets/m_avas.json

```diff
@@ -1 +1,87 @@
+"002":{"keywords":"Avengers, Martial Artist, S.H.I.E.L.D, Thunderbolts","name":"Hawkeye","points":"85"},
+"003":{"keywords":"Builders, Robot","name":"Aleph","points":"140/75"},
+"004":{"keywords":"Sons, of the Serpent","name":"Son of the Serpent","points":"33"},
+"005":{"keywords":"Armor, Initiative, New Warriors","name":"Scarlet Spider","points":"60"},
+"006":{"keywords":"Initiative","name":"Hazmat","points":"86"},
+"007":{"keywords":"Brute, Iniative","name":"Mettle","points":"88"},
+"008":{"keywords":"Animal, Avengers, Initiative","name":"Reptil","points":"70"},
+"009":{"keywords":"Armor, Initiative, New Warriors, Reporter","name":"Turbo","points":"50"},
+"010":{"keywords":"Avengers, Martial Artist, The Hand","name":"Echo","points":"96"},
+"011":{"keywords":"Avengers, Mystical, Rangers","name":"Firebird","points":"105"},
+"012":{"keywords":"Avengers, Defenders, Lady Liberators, Marvel Diva, Mystical","name":"Hellcat","points":"73"},
+"013":{"keywords":"Avengers, Celebrity, Initiative","name":"Triathlon","points":"60"},
+"013b":{"keywords":"Agents of Atlas, Avengers, Initiative","name":"3-D Man","points":"90"},
+"014":{"keywords":"Avengers, Initiative, New Warriors","name":"Justice","points":""},
+"015":{"keywords":"Avengers, Rangers","name":"Living Lightning","points":"80"},
+"016":{"keywords":"Avengers, Brute, Initiative, New Warriors","name":"Rage","points":"74"},
+"017":{"keywords":"Avengers, Heros for Hire, S.H.I.E.L.D","name":"Falcon","points":"75"},
+"018":{"keywords":"Martial Artist, Spy, Young Avengers","name":"Kate Bishop","points":"86"},
+"019":{"keywords":"Pilot, Pirate","name":"Cammi","points":"51"},
+"020":{"keywords":"Initiative, Sentinel","name":"Juston Seyfert","points":"40"},
+"021":{"keywords":"Asgardian, Initiative","name":"Thunderstrike","points":"98"},
+"022":{"keywords":"Initiative, Kree, New Warriors","name":"Ultragirl","points":"98"},
+"023":{"keywords":"Initiative, Spy","name":"Veil","points":"47"},
+"024":{"keywords":"Avengers, Reavengers","name":"D-Man","points":"84/"},
+"025":{"keywords":"Avengers, Eternal","name":"Forgotten One","points":"216/110"},
+"026":{"keywords":"Animal, Avengers, Deity, Warrior","name":"Silverclaw","points":"61"},
+"027":{"keywords":"Initiative, Mystical, Skulls","name":"Hybrid","points":"96"},
+"028":{"keywords":"Avengers, Freedom Force, Mystical, Omega Flight, Spider-Man Family","name":"Spider-Woman","points":"7"},
+"029":{"keywords":"Avengers, Batroc's Brigade, Lethal, Manarin's Minions, Masters of Evil","name":"Swordsman","points":"85"},
+"030":{"keywords":"Batroc's Brigade, HYDRA, Lethal, Legion, Mandarin's Minions, Masters of Evil, Zodiac","name":"Whirlwind","points":""},
+"031a":{"keywords":"Dark Avengers","name":"Moonstone","points":"126"},
+"031b":{"Keywords":"Exiles, Ruler, Squadron Supreme, Warrior","name",:"Power Princess","points":"150"},
```

Comment:
There can't be a `,` between a `key` and a `value`.
See `"name",:"Power Princess"`